src/Admin/ReportAdmin.php

```diff
@@ -255,10 +255,6 @@ protected function configureListFields(ListMapper $listMapper)
                 'label' => 'Date du signalement',
                 'format' => self::DATE_FORMAT,
             ])
-            ->add('details', null, [
```

Comment:
why do you remove this column? it seems to me that this isn't what is requested, no?

Reply:
Actually, it was requested. We will show this information only on the show page and not on the list ont.